images/ose-kubevirt-csi-driver-rhel8.yml

```diff
@@ -31,15 +31,3 @@ owners:
 - rgolan@redhat.com
 - nargaman@redhat.com
 - dvossel@redhat.com
-alternative_upstream:
```

Comment:
Change looks fine, but I think we should request for a new distgit without the `rhel8` part in the name, and update the filename accordingly

Reply:
I can pull from [containers/ose-kubevirt-csi-driver-rhel8](https://pkgs.devel.redhat.com/cgit/containers/ose-kubevirt-csi-driver-rhel8) and push to a new rhaos-4.16-rhel-9 branch on [containers/ose-kubevirt-csi-driver](https://pkgs.devel.redhat.com/cgit/containers/ose-kubevirt-csi-driver) if that works